File: engine/model/src/main/scala/net/server/channel/packet/interaction/SetItemsPlayerInteractionPacket.scala
package net.server.channel.packet.interaction

class SetItemsPlayerInteractionPacket(private var _mode: Byte, private var _slotType: Byte, private var _position: Short, private var _quantity: Short, private var _targetSlot: Byte) extends BasePlayerInteractionPacket(_mode = _mode) {
  def slotType: Byte = _slotType

  def position: Short = _position

  def quantity: Short = _quantity

  def targetSlot: Byte = _targetSlot
}

File: engine/engine-model/src/main/java/net/server/channel/packet/interaction/SetItemsPlayerInteractionPacket.java
package net.server.channel.packet.interaction;

public class SetItemsPlayerInteractionPacket extends BasePlayerInteractionPacket {
   private final Byte slotType;

   private final Short position;

   private final Short quantity;

   private final Byte targetSlot;

   public SetItemsPlayerInteractionPacket(Byte mode, Byte slotType, Short position, Short quantity, Byte targetSlot) {
      super(mode);
      this.slotType = slotType;
      this.position = position;
      this.quantity = quantity;
      this.targetSlot = targetSlot;
   }

   public Byte slotType() {
      return slotType;
   }

   public Short position() {
      return position;
   }

   public Short quantity() {
      return quantity;
   }

   public Byte targetSlot() {
      return targetSlot;
   }
}
